### ISSUE TITLE
Stacktrace when following the Quickstart

### ISSUE THREAD

**Issue by UtahDave** [saltstack/pop#52]
I'm following the Quickstart and when I get to the part where we're adding configuration data I get a stacktrace.

https://pop.readthedocs.io/en/latest/tutorial/quickstart.html#adding-configuration-data

I add the content to `poppy/conf.py` and then update my `poppy/poppy/init.py` and then when I run `python3 ./run.py --help` I get the following stacktrace

```
[boucha@flaco poppy]$ python3 ./run.py --help
Traceback (most recent call last):
  File "./run.py", line 5, in <module>
    hub.pop.sub.add('poppy.poppy')
  File "/usr/local/lib/python3.6/dist-packages/pop/contract.py", line 131, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pop/mods/pop/sub.py", line 61, in add
    root._subs[subname]._sub_init(init)
  File "/usr/local/lib/python3.6/dist-packages/pop/hub.py", line 300, in _sub_init
    self._find_mod('init', match_only=True)
  File "/usr/local/lib/python3.6/dist-packages/pop/hub.py", line 331, in _find_mod
    self._load_item(iface, bname)
  File "/usr/local/lib/python3.6/dist-packages/pop/hub.py", line 365, in _load_item
    self._prep_mod(mod, iface, bname)
  File "/usr/local/lib/python3.6/dist-packages/pop/hub.py", line 406, in _prep_mod
    pop.loader.mod_init(self._hub, mod)
  File "/usr/local/lib/python3.6/dist-packages/pop/loader.py", line 206, in mod_init
    mod.__init__(hub)
  File "/home/boucha/code/poppy/poppy/poppy/init.py", line 2, in __init__
    hub.pop.conf.integrate(['poppy'], loader='yaml', cli='poppy', roots=True)
  File "/usr/local/lib/python3.6/dist-packages/pop/contract.py", line 131, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pop/mods/pop/conf.py", line 26, in integrate
    logs=logs)
  File "/usr/local/lib/python3.6/dist-packages/pop/contract.py", line 131, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/pop/mods/conf/integrate.py", line 88, in load
    confs[imp].update(copy.deepcopy(cmod.CLI_CONFIG))
KeyError: 'poppy'
```

I'm not sure if I've gotten something wrong or if there's an actual bug

**Comment by UtahDave** [comment]
OK, after opening this issue and looking at further documentation here: https://pop.readthedocs.io/en/latest/topics/conf.html#conf-overview

I think I found the problem.

I think the following:


```
CLI_CONFIG = {
        'addr': {
            'options': ['-a'],
            'default': '127.0.0.1',
            'help': 'The address to present the rpc server on',
            },
        'port': {
            'options': ['-p'],
            'default': 8888,
            'help': 'The port to bind to',
            },
        }
```

should be:

```
CONFIG = {
        'addr': {
            'options': ['-a'],
            'default': '127.0.0.1',
            'help': 'The address to present the rpc server on',
            },
        'port': {
            'options': ['-p'],
            'default': 8888,
            'help': 'The port to bind to',
            },
        }
```

When I change `CLI_CONFIG` to `CONFIG` then the example works for me.

I'll open up a PR with the fix

**Comment by UtahDave** [comment]
OK, I think I figured out what's going on here.

Apparently `CONFIG` is required in the `poppy/conf.py` file.

I ran pop-seed to create a new test application and looked at the `project/conf.py` I noticed that it has these contents by default:

```
CLI_CONFIG = {}
CONFIG = {}
GLOBAL = {}
SUBS = {}
DYNE = {}
```

So I simply added `CONFIG = {}` to my `poppy/conf.py` and now I don't get the stacktrace and my application works.

Here's my final `poppy/conf.py`

```
CLI_CONFIG = {
        'addr': {
            'options': ['-a'],
            'default': '127.0.0.1',
            'help': 'The address to present the rpc server on',
            },
        'port': {
            'options': ['-p'],
            'default': 8888,
            'help': 'The port to bind to',
            },
        }

CONFIG = {}
```

So for now we can add `CONFIG = {}` to the Quickstart example so that the application doesn't barf. I'll create a PR for that.

Do we want to update pop so that it doesn't stacktrace if `CONFIG` is missing? I'll let you decide how best to handle that.
